app/account/serializers.py

```diff
@@ -0,0 +1,84 @@
+from pydantic import constr, validator, root_validator
+
+from models import Account
+
+from utils.exceptions import SerializerValidationError
+from utils.serializers import BaseSerializer, BaseModelSerializer, serializer_data_type
+from utils.data_process import make_password
+
+
+class RegisterUserSerializer(BaseSerializer):
+    """ Serializer for register users """
+
+    username: constr(max_length=100)  # type: ignore
+    password: constr(max_length=500)  # type: ignore
+    repeat_password: constr(max_length=500)  # type: ignore
+    email: constr(max_length=100)  # type: ignore
+    first_name: constr(max_length=100)  # type: ignore
+    last_name: constr(max_length=100)  # type: ignore
+
+    @validator('username')
+    def validate_username(cls, value: str) -> str:
+        from models import Account
+
+        accounts = Account.query.filter_by(username=value).all()
+        if any(accounts):
+            raise SerializerValidationError({'error': 'Account with this username already exist'})
+        return value
+
+    @validator('email')
+    def validate_email(cls, value: str) -> str:
+        from models import Account
```

Comment:
you have too many local imports. if that's because of circular dependencies, fix this by structuring your modules differently